support_bundle/get_support_bundle.sh

```diff
@@ -11,216 +11,396 @@ catch() {
+        echo "kubectl ${KUBE_OPTS} get ns";
+        echo "$(kubectl ${KUBE_OPTS} get ns)";
+    fi
+    
+    # If API key is supplied, collect streamSnap, Index settings, and fastPath settings
+    if [[ ! -z ${API_KEY} ]]; then
+        API_URL=$(kubectl ${KUBE_OPTS} get cm sysdigcloud-config -o yaml | grep -i api.url: | head -1 | awk '{print$2}')
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/license" >> ${LOG_DIR}/license.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customer/1/streamsnapSettings" >> ${LOG_DIR}/streamSnap_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customers/1/snapshotSettings" >> ${LOG_DIR}/snapshot_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customer/1/fastPathSettings" >> ${LOG_DIR}/fastPath_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customer/1/indexSettings" >> ${LOG_DIR}/index_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customer/1/planSettings" >> ${LOG_DIR}/plan_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/admin/customer/1/dataRetentionSettings" >> ${LOG_DIR}/dataRetention_settings.json
+        curl -ks -H "Authorization: Bearer ${API_KEY}" -H "Content-Type: application/json" "${API_URL}/api/agents/connected" >> ${LOG_DIR}/agents-connected.json
+    
+        TO_EPOCH_TIME=$(gnudate -d "$(gnudate +%H):00:00" +%s)
+        FROM_EPOCH_TIME=$((TO_EPOCH_TIME-86400))
+        METRICS=("syscall.count" "dragent.analyzer.sr" "container.count" "dragent.analyzer.n_drops_buffer" "dragent.analyzer.n_evts")
+        DEFAULT_SEGMENT="host.hostName"
+        SYSCALL_SEGMENTS=("host.hostName" "proc.name")
+    
+        mkdir -p ${LOG_DIR}/metrics
+        for metric in ${METRICS[@]}; do
+            if [ "${metric}" == "syscall.count" ]; then
+                for segment in ${SYSCALL_SEGMENTS[@]}; do
+                    get_metrics "${metric}" "${segment}"
+                done
+            else
+                get_metrics "${metric}" "${DEFAULT_SEGMENT}"
+            fi
+        done
+    
+        get_agent_version_metric_limits
+    fi
+    
+    # Configure kubectl command if labels are set
+    if [[ -z ${LABELS} ]]; then
+        SYSDIGCLOUD_PODS=$(kubectl ${KUBE_OPTS} get pods | awk '{ print $1 }' | grep -v NAME)
+    else
+        SYSDIGCLOUD_PODS=$(kubectl ${KUBE_OPTS} -l "role in (${LABELS})" get pods | awk '{ print $1 }' | grep -v NAME)
+    fi
+    
+    echo "Using namespace ${NAMESPACE}";
```

Comment:
nit, but please remove trailing `;`

Reply:
@mbreitung fixed in new commit [https://github.com/draios/sysdig-cloud-scripts/pull/191/commits/2e60d77bf2a5c914ce5ec79abe1bbb7d57a0c984](https://github.com/draios/sysdig-cloud-scripts/pull/191/commits/2e60d77bf2a5c914ce5ec79abe1bbb7d57a0c984)